src/main/java/se/sundsvall/supportmanagement/service/AccessMapperService.java

```diff
@@ -24,9 +32,14 @@ public AccessMapperService(final AccessMapperClient accessMapperClient, final Me
 		this.metadataService = metadataService;
 	}
 
-	// Add cache
+	@Cacheable(value = ACCESSIBLE_LABELS_CACHE_NAME,
```

Comment:
Missing test that actually verifies that cache is used. All test just verifies that annotations exists.